features/docker.feature

```diff
@@ -74,3 +74,92 @@ Feature: Build docker images with pro services
            | mantic  | lxd-vm       | xenial            | [ esm-infra ]  | curl              | esm                  |
            | mantic  | lxd-vm       | bionic            | [ fips ]       | openssl           | fips                 |
            | mantic  | lxd-vm       | focal             | [ esm-apps ]   | hello             | esm                  |
+
```

Comment:
I think we need to drop the `@uses.config.contract_token` from the Feature definition, otherwise we will skip the following tests

Reply:
Good catch! I'll add it to only the scenarios that need it within this feature.

Reply:
@lucasmoura updated!